librosa/core/notation.py

```diff
@@ -472,6 +474,61 @@ def list_thaat() -> List[str]:
     return list(THAAT_MAP.keys())
 
 
+def note_to_degree(key: str) -> int:
```

Comment:
Is this function meant to be user-facing or purely internal?

If the former, we'll need to expand out the docstring following numpydoc conventions, and add the function to `__all__` and `__init__.pyi`.

If the latter, we should rename this to `__note_to_degree` to convey that the function is intended as private.

Reply:
I intended it only as a helper function, but if you think there could be some use to it to the user, I could make the necessary changes.

Reply:
@bmcfee Let me know what you think about this. I'm happy to do either, but I lean towards keeping the functions private.

Reply:
I think keeping them private makes sense.

Reply:
Ok, sounds good! I pushed that change to the branch.